src/runtime/pkg/katautils/config.go

```diff
@@ -61,6 +61,12 @@ type tomlConfig struct {
 	Runtime    runtime
 	Factory    factory
 	Netmon     netmon
+	Image      image
```

Comment:
Maybe `Images` or `ImageDownload`?

Reply:
Image is correct, this is the name of the toml's section